email/1.3/modules/ROOT/edits/email-send.adoc

```diff
@@ -0,0 +1,363 @@
+= Send Emails with Email Connector Examples - Mule 4
+:keywords: email, connector, configuration, smtp, send, smtps
+:page-aliases: connectors::email/email-send.adoc
+
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
+
+* <<send-email,Send Emails>> +
+Configure the Email Connector *Send* operation to send emails over an SMTPS server.
+
+* <<send-attachments,Send an Attachment>> +
+Configure the File Connector *Read* operation to read a JSON file and the Email Connector *Send* operation to send the file as an attachment over an SMTP server.
+
+* <<send-attachments,Send Multiple Attachments>> +
+Configure *Set Variable* components to define multiple media type attachments, and send these attachments using the Email Connector *Send* operation. Then log the result message using a *Logger* component.
+
+[[send-emails]]
+== Send Emails
+
+The following example illustrates how to send emails over the SMTPS server. The flow initiates with a *Scheduler* component, and then the *Send* operation sends the email.
+The SMTPS connection type enables SSL or TLS encryption and sends encrypted messages over the secured version of the SMTP server.
+
+The following screenshot shows the app flow for this example:
+
+.Email Connector Send messages over SMTPS
+image::email-send-flow.png[Email Connector Send messages over SMTPS flow]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, select the *Scheduler* component and drag it onto the canvas. +
+This source initiates a flow when a time-based condition is met.
+. Drag the Email *Send* operation to the right of the *Scheduler* source. +
+. On the *Send* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to configure a global element for the operation.
```

Comment:
"on the screen"?
GLOBAL edit in other files in this set, re. "on" vs. "in"

Reply:
Actually, I am going to review this document and add what I think is relevant to our style ref, so stand by....:
https://developers.google.com/style/ui-elements